src/VisualStudio/Core/Impl/Options/AbstractOptionPageControl.cs

```diff
@@ -72,9 +83,9 @@ protected void BindToOption(CheckBox checkbox, PerLanguageOption<bool> optionKey
         {
             var binding = new Binding()
             {
-                Source = new PerLanguageOptionBinding<bool>(OptionService, optionKey, languageName),
+                Source = new PerLanguageOptionBinding<bool>(OptionStore, optionKey, languageName),
                 Path = new PropertyPath("Value"),
-                UpdateSourceTrigger = UpdateSourceTrigger.Explicit
+                UpdateSourceTrigger = UpdateSourceTrigger.Default
```

Comment:
Can we just delete all the UpdateSourceTriggers setting since we're now setting it to default?